src/main/java/pvs/app/service/data/TrelloData.java

```diff
@@ -0,0 +1,60 @@
+package pvs.app.service.data;
+
+import java.util.ArrayList;
+
+public class TrelloData {
+    private final ArrayList<TrelloList> lanes;
+
+    public TrelloData(){
+        this.lanes = new ArrayList<>();
+    }
+
+    public TrelloList createList(String id, String title, String label, int width) {
```

Comment:
Suggest move this creation function into its self class, which is the `Lane` class.

And also suggest rename this function to `createLane`.
```suggestion
    public TrelloList createLane(String id, String title, String label, int width) {
```